client/src/views/Simulation/components/ProvenanceGraph/ProvenanceGraph.vue

```diff
@@ -12,6 +12,7 @@
           {{ layout.name }}
         </button>
       </div>
+      <h7>{{sessionMetadata}}</h7>
```

Comment:
`<h7>` does not exist, `<h6>` is the limit.

Reply:
Yep, I'm getting an error.

Reply:
Thanks, I've fixed it :)